ocw_data_parser/ocw_data_parser.py

```diff
@@ -546,6 +547,15 @@ def generate_parsed_json(self):
             "master_course_number": master_course.split(".")[1]
             if master_course
             else "",
+            "new_course_numbers": [
+                {
+                    key.replace("aka_", ""): value
```

Comment:
Can we do this replacment only if `aka_` is a prefix? This could cause problems if `aka_` appears elsewhere in the string